src/components/menu/API.tsx

```diff
@@ -0,0 +1,128 @@
+import * as React from "react";
+import {Trans, withTranslation, WithTranslation} from "react-i18next";
+import {IAlert} from "../../types";
+
+
+interface IState {
+    alerts: IAlert[];
+}
+
+class API extends React.Component<WithTranslation, IState> {
+
+    state = {alerts: []};
+
+    public render() {
+    
+        return (
+            <div>
+                <div className="content box" id="API">
+                    <h3><Trans>API</Trans></h3>
+                        <h4>Preset</h4>
+                        <h5>Preset</h5>
+                            <p>Request the selected preset.</p>
+                            <dl>
+                                <dt>Endpoint</dt>
+                                <dd>/api/preset</dd>   
+                                <dt>Request Parameters</dt>
+                                <dd>id</dd>
+                                <dt>Example Request</dt>
+                                <dd>https://aoe2cm.net/api/preset/simple</dd>
+                                <dd>
+                                <ul>
+                                    <li>TECHNICAL = 0</li>
+                                    <li>AOK = 1</li>
+                                    <li>CONQUERORS = 2</li>
+                                    <li>FORGOTTEN = 3</li>
+                                    <li>AFRICAN_KINGDOMS = 4</li>
+                                    <li>RISE_OF_RAJAS = 5</li>
+                                    <li>DEFINITIVE_EDITION = 6</li>
+                                    <li>LORDS_OF_THE_WEST = 7</li>
+                                    </ul>
```

Comment:
Not sure what this enum has to do with this endpoint :thinking:

Reply:
when i ask for the draft, the civ has a "gameVersion" which seems to be this number. I added it here to have it somewhere. I think i would prefer /api/strings, similar to aoe2.net for this

Reply:
Well, ideally you would have documentation for all the used classes (Preset, Draft, etc.), but for now people have to check either the sources or the actual data and infer them from there. You should never need to actually build civs yourself, they are always encoded in the civilisationArray